Assets/GothicVR/Scripts/Manager/ConstantsManager.cs

```diff
@@ -19,6 +19,7 @@ public class ConstantsManager : SingletonBehaviour<ConstantsManager>
         //Collection of PlayerPref entries for settings
         public string moveSpeedPlayerPref = "MoveSpeed";
         public string turnSettingPlayerPref = "TurnSetting";
+        public string turnAngelPlayerPref = "TurnAngelSetting";
```

Comment:
```suggestion
        public string turnAnglePlayerPref = "TurnAngleSetting";
```

Angel == 🎣 in German
Angle == 📐in English
;-)